data-es6.js

```diff
@@ -16864,9 +16864,12 @@ exports.tests = [
     {
       name: 'duplicate identifier',
       exec: function(){/*
-        var d = function d([d]) { return d };
-        if (d([true]) !== true) return false;
-
+        try {
+          eval('var d = function d([d]) { return d };if (d([true]) !== true) return false;');
+        } catch (e) {
+          return !(e instanceof SyntaxError);
```

Comment:
```suggestion
          return e instanceof SyntaxError;
```
since this should be a syntax error, no?

Reply:
No it should not be a syntax error. In
```
var d = function d([d]) { return d };
```
Safari thought the parameter `d` is duplicated, but the other `d` is just the function name, not an actual parameter.
See also https://bugs.webkit.org/show_bug.cgi?id=220517.

Reply:
ahhhh ok, thanks, that clarifies things.